cassiopeia/tools/fitness_estimator/_jungle/environment.yml

```diff
@@ -0,0 +1,95 @@
+name: jungle
```

Comment:
do we need to include Felix's environment.yml here? are there any dependencies here that are needed that we don't have in the regular Cassiopeia requirements?